docs/azure/sdk/unit-testing-mocking.md

```diff
@@ -231,6 +231,50 @@ var finder = new AboutToExpireSecretFinder(TimeSpan.FromDays(2), secretClient);
 
 This approach is useful when you would like to consolidate the dependency creation and share the client between multiple consuming classes.
 
+## Explore the extension methods in Azure Resource Manager SDKs
```

Comment:
```suggestion
## Understand Azure Resource Manager clients
```

We should use the same form as the "Understand service clients" here; that means, we'll need an overview.

It is also important to note that the "Azure SDK" is the all-up product name.  Individual packages are "client libraries" or "management libraries" - we've been asked not to refer to them as "SDKs".

Reply:
Thank you! Fixed